modules/oe_content_event/modules/oe_content_event_event_programme/src/Entity/ProgrammeItem.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_event_event_programme\Entity;
+
+use Drupal\oe_content_entity\Entity\CorporateEntityBase;
+
+/**
+ * Defines the Programme Item entity.
+ *
+ * @ingroup oe_content_event_event_programme
+ *
+ * @ContentEntityType(
+ *   id = "event_programme",
```

Comment:
```suggestion
 *   id = "oe_event_programme",
```
Even @ademarco defined machine name as `event_programme`, we have to prefix this library entities with prefix `oe_`